solidity/random-beacon/tasks/staking.ts

```diff
@@ -0,0 +1,110 @@
+import { task, types } from "hardhat/config"
+
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+
+task(
+  "staking",
```

Comment:
I think "stake" suggests that it calls only a `stake()`. I wanted to name it a more general "staking" because we also invoke some functions around staking.